wrapper/cpp/main.go

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache v2.0 License.
+
+//
+// This file contains wrapper function calls that c++ component
+// can leverage to call into MocStack
+//
+
+package main

```

Comment:
Is there a way we could have some directory structure as go services, so we can expand adding more apis.

Reply:
I dont know how to implement file structures. What I read is all exported functions needs to be coded up in the go file where package is main and there is main() function. Do you know how to implement it? I think what you are suggesting is to have, for example, EncryptData() under wrapper\cpp\security\keyvault\key\ folder.

If your team can help provide guidance how to do that, I can open a task to track this. At this point we are very close to code complete, hence I prefer to get this change in to unblock our team from integrating with attestation agent.

Reply:
At this point we dont have details. Lets go with this and refactor it later.

Reply:
Thanks Madhan. All checks are passing now, is it okay to merge?